Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Create_a_security_pre-rule_for_EDL.yml

```diff
@@ -0,0 +1,610 @@
+  "6":
+    id: "6"
+    taskid: 961d0261-a4e6-4ebf-8685-2191b876bc49
+    type: regular
+    task:
+      id: 961d0261-a4e6-4ebf-8685-2191b876bc49
+      version: -1
+      name: 'Get EDL object '
+      description: Lists all external dynamic lists.
+      script: '|||prisma-sase-external-dynamic-list-list'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      folder:
+        complex:
+          root: inputs.Folder
+      name:
+        complex:
+          root: inputs.EDLObjectName
+      tsg_id:
+        complex:
+          root: inputs.TSGID
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 940,
+          "y": -40
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: e9cff0a1-05a2-47b0-8643-1b58f6a0e912
+    type: condition
+    task:
+      id: e9cff0a1-05a2-47b0-8643-1b58f6a0e912
+      version: -1
+      name: Was there a returned object?
+      description: Was there a returned object?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "10"
+      "yes":
+      - "9"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: PrismaSase.ExternalDynamicList.id
+            iscontext: true
+          right:
+            value: {}
+      - - operator: isEqualString
+          left:
+            value:
+              simple: PrismaSase.ExternalDynamicList.source
+            iscontext: true
+          right:
+            value:
+              simple: inputs.EDLObjectName
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 940,
+          "y": 130
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: 5b777876-447c-4a3c-8d28-204661342395
+    type: playbook
+    task:
+      id: 5b777876-447c-4a3c-8d28-204661342395
+      version: -1
+      name: Prisma SASE - Create or Edit EDL object
+      playbookName: Prisma SASE - Create or Edit EDL object
+      type: playbook
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "10"
+    scriptarguments:
+      AutoCommit:
+        simple: Else
+      EDLObjectType:
+        complex:
+          root: inputs.EDLObjectType
+      Folder:
+        complex:
+          root: inputs.Folder
+      Name:
+        simple: EDL Block Indicators
+      SourceURL:
+        complex:
+          root: inputs.URL
+      TSGID:
```

Comment:
missing playbook inputs for 
![image](https://github.com/demisto/content/assets/61933087/e263a005-b5b8-4407-9126-43e1ab524683)

Reply:
Done
The idea was not really creating the pre-defined here ( they are limited with the options and most of them are already in the UI). but I still added those inputs just in case.